packages/browser/src/plugins/remote-loader/index.ts

```diff
@@ -79,7 +79,21 @@ export class ActionDestination implements DestinationPlugin {
         transformedContext = await this.transform(ctx)
       }
 
-      await this.action[methodName]!(transformedContext)
+      try {
+        ctx.stats.increment('analytics_js.action_plugin.invoke', 1, [
+          `method:${methodName}`,
+          `action_plugin_name:${this.action.name}`,
+        ])
+
+        await this.action[methodName]!(transformedContext)
+      } catch (error) {
+        ctx.stats.increment('analytics_js.action_plugin.invoke.error', 1, [
```

Comment:
We are repeating this string a few times. Can we save bytes through a helper fn?

Reply:
gzip is extremely good at handling repeating strings, I've verified it - sometimes placing the strings inline yields a smaller size than storing it in a variable.

Reply:
Even so, I still think DRY would be an improvement here for the usual reasons?

Reply:
It's like this all across the codebase, we should refactor them all at once so that the declaration and usage is standardized all across, and not fragment it even further.

Reply:
Fair

Reply:
I see the benefit of storing in a variable more that we have 1 place to update/less chance of a typo/forgetting to update the value if we change it in the future. Won't block, but that would be my general guidance when we've got a string value that acts as a key in downstream.